nixos/modules/services/audio/roon-server.nix

```diff
@@ -61,8 +61,8 @@ in {
     };
 
     
-    users.groups."${cfg.group}" = {};
-    users.users."${cfg.user}" =
+    users.groups.${cfg.group} = {};
```

Comment:
this is not correct

Reply:
> this is not correct

Why?

Reply:
Ah, nevermind. I thought that was actually invalid syntax but it works.